lib/sessionManager/types.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * This interfaces provides the contract that an session management utility must
+ * satisfiy in order to work with this SDK, please vist the example provided in the
+ * README, to understand how this works.
+ */
+type Awaitable<T> = Promise<T>;
+
+export enum StorageKeys {
+  accessToken = "accessToken",
+  idToken = "idToken",
+  refreshToken = "refreshToken",
+}
+
+export type StorageSettingsType = {
```

Comment:
What is the plan for this type 😃

Reply:
there was a mistake in the `sessionManager/index.ts` it should have been using this type 👍